examples/contracts/src/lib.rs

```diff
@@ -320,14 +314,15 @@ mod tests {
         // ANCHOR_END: simulate
         let response = contract_instance.mint_coins(1_000_000).call().await?;
         // ANCHOR: variable_outputs
-        let address = wallet.address();
+        // TODO: Enable test
```

Comment:
Could you elaborate on why these tests are failing?

Reply:
@MujkicA https://github.com/FuelLabs/fuels-rs/pull/546#pullrequestreview-1089040916 should answer! TLDR: stdlib needs to support some new stuff.